app/assets/javascripts/application.js

```diff
@@ -23,4 +23,12 @@ $(function(){ $(document).foundation(); });
 
 $(document).ready(function() {
   $("#user_phone").mask("(999) 999-99-99");
+  $('a').removeClass('currentPage')
 });
+
+function markLink(text) {
```

Comment:
nope, this is just for marking 'Profile', for example, link in navigation bar with some color when the user is on his profile's page.